### PR TITLE
Default status on WorkflowLevel2

### PR DESCRIPTION
**Pull request by ralfzen**
## Purpose
With the new WFL-status-filter all WFLs without any status are hidden.

## Approach
Disable the blank option and add a migration for existing WFLs without a status.

### Further Info
https://humanitec.atlassian.net/browse/CU-735

### CODE REVIEW

File: workflow/models.py [hunk 1]
```diff
@@ -307,6 +307,14 @@ def __str__(self):
             return self.name
 
 
+def _get_default_statuslevel():
```
Comment (ralfzen) [inline workflow/models.py]: @docktorrr Do you have a better idea, about where to put this method?

Comment (docktorrr) [reply workflow/models.py]: I would simply put it to `WorkflowLevel1.save()`

**Comment by ralfzen** [comment]
@docktorrr That might create problems with create-operations, where the `save()` is not called, like `bulk_create`. I think the `default=` option here is cleaner.

**Comment by docktorrr** [comment]
@ralfzen Yes, you are right.

But now I'm thinking why do you need model for the status instead of pre-defined constants?
You use hardcoded constant `project_request` in the code to create status object, so why just not create constants for the statuses?

**Pull request merged**